source/_integrations/matter.markdown

```diff
@@ -255,6 +255,19 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
    - For example, if you shared it with Google Home, it lists **Google LLC**.
      ![image](/images/integrations/matter/matter_share-device-with-other-platform.png)
 
+## Removing a device from a network
+
+Follow these steps if you want to remove a device from a particular network.
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %} and on the **Matter (BETA)** integration card, select **Devices**.
+2. From the list of devices, select the device you want to remove from a network.
+3. In the **Device info** section, next to **Share device**, select the three dot menu. Then, select **Manage fabrics**.
+4. From the list, remove the network of interest. 
+   - If you want to remove Apple Home, also remove the Apple Keychain entry.
+   ![image](/images/integrations/matter/matter-remove-from-network.png)
+
+
```

Comment:
Another note could be that if you want to remove the device from the home Assistant fabric, you need to press remove from the 3 dots context menu

Reply:
added a step